src/test/resources/com/puppycrawl/tools/checkstyle/meta/javadocmetadatascraper/InputJavadocMetadataScraperAbstractSuperCheck.java

```diff
@@ -12,9 +12,9 @@
 
 import com.puppycrawl.tools.checkstyle.FileStatefulCheck;
 import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
-import com.puppycrawl.tools.checkstyle.api.DetailAST;
-import com.puppycrawl.tools.checkstyle.api.TokenTypes;
```

Comment:
Though this is the meta scapper, I wasn't seeing any uses of these imports so I removed them because of their connection to this issue.
Let me know if there is a reason for these.

Reply:
Please recheck git history.

Reply:
@romani When I rechecked one file `InputJavadocMetadataScraperNoCodeInFileCheck`, the file had been like that since the beginning.
> Pull #10827: reworks JavadocMetadataScrapperTest to make more sense

This is more me moving files around.

> Issue https://github.com/checkstyle/checkstyle/issues/8771: Improve code coverage for metadata generator classes

This is where the file really started and it is the same. I didn't see anything in PRs, and this issue was for code coverage. I can only assume we wanted more of a representation of the Check without copying the check's full source.

Reply:
Ok, let's merge this PR